scripts/gulp/scripts.js

```diff
@@ -26,54 +24,53 @@ const COMMON_UGLIFY_CONFIG = new UglifyWebpackPlugin( {
   }
 } );
 
-// TODO: Add a webpack-config file to handle sharing of redundant webpack
-//       configurations. Also, add a production and dev flag to generate
-//       a minified and un-minified version of the assets.
+/* TODO: Add a webpack-config file to handle sharing of redundant webpack
+   configurations. Also, add a production and dev flag to generate
+   a minified and un-minified version of the assets. */
 
 // Compile the master capital-framework.less file.
-gulp.task( 'scripts:cf', () => {
-  return gulp.src('./src/capital-framework.js')
-    .pipe( webpackStream( {
-      module: {
-        rules: [ {
-          use: [ {
-            loader: 'babel-loader?cacheDirectory=true',
-            options: {
-              presets: [ [ 'env', {
-                targets: {
-                  browsers: BROWSER_LIST.LAST_2_PLUS_IE_9_AND_UP
-                },
-                debug: true
-              } ] ]
-            }
-          } ]
+gulp.task( 'scripts:cf', () => gulp.src( './src/capital-framework.js' )
```

Comment:
Should these get a `TODO` to convert them to named functions like we've done elsewhere?

Reply:
Normally I'd say yes, but I can scoop them up in the gulp 4 PR. I won't forget these!